src/CTA.Rules.Update/WCFServicePort.cs

```diff
@@ -0,0 +1,314 @@
+                {
+                    var binding = keyValuePair.Key;
+                    var mode = keyValuePair.Value.Mode;
+                    var endpointAddress = keyValuePair.Value.EndpointAddress ?? String.Join("", "\\", binding.ToLower());
+
+                    if (binding == Constants.HttpProtocol)
+                    {
+                        endpointConfigs += String.Format(endpointConfigTemplate, serviceClassName, serviceInterfaceName, 
+                            "new BasicHttpBinding(BasicHttpSecurityMode." + mode + ")", endpointAddress);
+                    }
+                    else if(binding == Constants.NettcpProtocol)
+                    {
+                        endpointConfigs += String.Format(endpointConfigTemplate, serviceClassName, serviceInterfaceName, "new NetTcpBinding(SecurityMode." + mode + ")", endpointAddress);
+                    }
+                    else if (binding == Constants.WSHttpProtocol)
+                    {
+                        endpointConfigs += String.Format(endpointConfigTemplate, serviceClassName, serviceInterfaceName, "new WSHttpBinding(SecurityMode." + mode + ")", endpointAddress);
+                    }
+                    else if (binding == Constants.HttpsProtocol)
+                    {
+                        endpointConfigs += String.Format(endpointConfigTemplate, serviceClassName, serviceInterfaceName, "new BasicHttpsBinding(BasicHttpsSecurityMode." + mode + ")", endpointAddress);
+                    }
+                    else if (binding == Constants.NethttpProtocol)
+                    {
+                        endpointConfigs += String.Format(endpointConfigTemplate, serviceClassName, serviceInterfaceName, "new NetHttpBinding(BasicHttpSecurityMode." + mode + ")", endpointAddress);
+                    }
+                }
+
+                return startupFileContents.Replace(Constants.EndpointPlaceholder, endpointConfigs);
+            }
+        }
+
+
+        /// <summary>
+        /// Determines whether project config has <behaviors> tag
+        /// </summary>
+        /// <returns>Whether <behaviors> tag is present</returns>
+        public bool HasBehavioursTag()
+        {
+            var config = WebConfigManager.LoadWebConfigAsXDocument(_projectPath);
+
+            return config.ContainsElement(Constants.BehaviorsPath);
+        }
+
+        /// <summary>
+        /// For Behaviors Tag, add Comment specifying it is unsupported in CoreWCF.s
+        /// </summary>
+        /// <param name="filePath">Path to existing Startup.cs</param>
+        /// <returns>New Startup.cs contents with Comment</returns>
+        public static string HandleBehaviorsTag(string filePath)
+        {
+            var lines = File.ReadAllLines(filePath).ToList();
+            var newFileContents = new StringBuilder();
+
+            foreach (var line in lines)
+            {
+                newFileContents.Append(line + "\n");
```

Comment:
nit: Can replace `Append()` with `AppendLine()`. `AppendLine()` uses `Environment.NewLine` which defaults to the new line char native to the platform being used.